nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/users/SuggestUserEntries.java

```diff
@@ -266,7 +265,7 @@ public Blob run() {
     protected DocumentModelList searchUsers() {
         if (StringUtils.isBlank(prefix)) {
             // empty search term
-            return new DocumentModelListImpl();
+            return userManager.searchUsers(prefix);
```

Comment:
`userManager.searchUsers()` would be more logical

Reply:
If the API existed you mean?